packages/protocol/migrationsConfig.js

```diff
@@ -330,29 +331,53 @@ const NetworkConfigs = {
       frozen: false,
     },
     reserve: {
-      initialBalance: 100000000,
+      initialBalance: 100000000, // CELO
     },
   },
   baklava: {
-    blockchainParameters: {
-      minimumClientVersion: {
-        major: 0,
-        minor: 10,
-        patch: 0,
+    election: {
+      frozen: false,
```

Comment:
I know we don't want to have frozen elections, but IMO to keep baklava as close to Mainnet as possible, what do you think about keeping them all frozen and have a governance proposal to unfreeze? I know that it shouldn't practically make a different, but you never know?

Reply:
We want the elections enabled immediately so folks can start joining the network as validators. I would also like to avoid the operational overhead of running governance proposals right away if possible. Given that we've already tested the code for starting a network with elections disabled on both the latest instance of Baklava and Mainnet, it doesn't seem like the extra coverage will make enough of a difference to warrant the work required.